config-native/build.gradle

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+plugins {
+    id 'java'
+}
+
+description = 'Ballerina - Config Java Utils'
+
+dependencies {
+    compile group: 'org.ballerinalang', name: 'ballerina-lang', version: "${ballerinaLangVersion}"
+    compile group: 'org.ballerinalang', name: 'ballerina-config', version: "${ballerinaLangVersion}"
+    compile group: 'org.ballerinalang', name: 'ballerina-reflect', version: "${ballerinaLangVersion}"
+    compile group: 'org.ballerinalang', name: 'ballerina-runtime', version: "${ballerinaLangVersion}"
+    compile group: 'org.ballerinalang', name: 'ballerina-stringutils', version: "${ballerinaLangVersion}"
+    compile group: 'org.ballerinalang', name: 'ballerina-runtime-api', version: "${ballerinaLangVersion}"
+    compile group: 'org.ballerinalang', name: 'ballerina-cache', version: "${ballerinaLangVersion}"
+    compile group: 'org.slf4j', name: 'slf4j-jdk14', version: "${slf4jVersion}"
+    compile group: 'org.ballerinalang', name: 'ballerina-runtime', version: "${ballerinaLangVersion}"
+    compile group: 'org.ballerinalang', name: 'ballerina-tool', version: "${ballerinaLangVersion}"
+    compile group: 'org.ballerinalang', name: 'ballerina-java', version: "${ballerinaLangVersion}"
+    compile group: 'org.ballerinalang', name: 'ballerina-time', version: "${ballerinaLangVersion}"
+    compile group: 'org.ballerinalang', name: 'ballerina-system', version: "${ballerinaLangVersion}"
+    compile group: 'org.ballerinalang', name: 'ballerina-io', version: "${ballerinaLangVersion}"
+    compile group: 'org.ballerinalang', name: 'value', version: "${ballerinaLangVersion}"
+    compile group: 'org.ballerinalang', name: 'typedesc', version: "${ballerinaLangVersion}"
+    compile group: 'org.ballerinalang', name: 'annotations', version: "${ballerinaLangVersion}"
+    compile group: 'org.ballerinalang', name: 'toml-parser', version: "${ballerinaTomlParserVersion}"
+    testCompile group: 'org.testng', name: 'testng', version: '7.3.0'
```

Comment:
Shall we parameterize this version too?

Reply:
Actually we don't need this anymore. Since all the java tests were moved out to bvm/ballerina-config. I'll remove this.